include/swift/SIL/OwnershipUtils.h

```diff
@@ -461,19 +461,9 @@ struct BorrowingOperand {
     llvm_unreachable("Covered switch isn't covered?!");
   }
 
-  /// Visit all of the "results" of the user of this operand that are borrow
-  /// scope introducers for the specific scope that this borrow scope operand
-  /// summarizes.
-  ///
-  /// Precondition: hasBorrowIntroducingUser() is true
-  ///
-  /// Returns false and early exits if \p visitor returns false.
-  bool visitBorrowIntroducingUserResults(
-      function_ref<bool(BorrowedValue)> visitor) const;
-
   /// If this operand's user has a single borrowed value result return a
   /// valid BorrowedValue instance.
-  BorrowedValue getBorrowIntroducingUserResult();
+  BorrowedValue getBorrowIntroducingUserResult() const;
```

Comment:
> `If this operand's user has a single borrowed`

I suggest removing the word "single" because it implies that multiple results are possible and that the API returns an invalid value in that case.